### PR TITLE
Update UG with error boxes

### PR DESCRIPTION
**Pull request by tjch-o**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -404,14 +404,20 @@ Format: `link ev/EVENT_NAME c/CONTACT_NAME [c/MORE_CONTACT_NAMES]`
...
 * `link ev/NUS Career Fest c/Alice Black` links `Alice Black` to the event `NUS Career Fest` if `Alice Black` is not linked to `NUS Career Fest`.
 
 <div style="page-break-after: always;"></div>
 
-### Unlinking contacts from an event: `unlink`
+<box type="warning" style="background-color: #C73852; color: white">
+<span slot="icon" style="color: white;"><md>:fas-close:</md></span>
+
+**Errors:**
+If any of the contacts or events keyed in are not valid and do not exist in JobFestGo, an error will be thrown.
```
Comment (Khaleelur-Rahman) [inline docs/UserGuide.md:417]: Typo: `event` and not `events`

File: docs/UserGuide.md [hunk 2]
```diff
@@ -420,12 +426,19 @@ Unlinks specified contacts from the specified event.
...
- must be currently linked. Otherwise, an error will be raised.
 
 Examples:
 * `unlink ev/NUS Career Fest c/Alice Black` unlinks `Alice Black` from the event `NUS Career Fest` if `Alice Black` is linked to `NUS Career Fest`.
 
+<box type="warning" style="background-color: #C73852; color: white">
+<span slot="icon" style="color: white;"><md>:fas-close:</md></span>
+
+**Errors:**
+Similar to [`link`](#linking-contacts-to-an-event-link), if any of the contacts or events keyed in are not valid and do not exist in JobFestGo, an error will be thrown.
```
Comment (Khaleelur-Rahman) [inline docs/UserGuide.md:437]: Similar typo as above

File: docs/UserGuide.md [hunk 3]
```diff
@@ -500,31 +519,48 @@ Format: `mark_task td/TASK_DESCRIPTION ev/EVENT_NAME`
...
 Examples:
-* `mark_task td/Book Venue ev/NUS Career Fair 2023` marks the task `Book Venue` from the task list of the event `NUS Career Fair 2023` as completed.<br><br>
+* `mark_task td/Book Venue ev/NUS Career Fair 2023` marks the task `Book Venue` from the task list of the event `NUS Career Fair 2023` as completed.
+
+<box type="warning" style="background-color: #C73852; color: white">
+<span slot="icon" style="color: white;"><md>:fas-close:</md></span>
+
+**Errors:**
+If the specified event does not exist or the event does not have the specified task, an error will be thrown.
+If such a situation happens, you may double-check the task description and the event name and re-enter valid inputs.
```
Comment (Khaleelur-Rahman) [inline docs/UserGuide.md:531]: Similar consideration as previous comment.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -500,31 +519,48 @@ Format: `mark_task td/TASK_DESCRIPTION ev/EVENT_NAME`
...
 
 Examples:
 * `unmark_task td/Book Venue ev/NUS Career Fair 2023` marks the task `Book Venue` from the task list of the event `NUS Career Fair 2023` as not completed.
 
+<box type="warning" style="background-color: #C73852; color: white">
+<span slot="icon" style="color: white;"><md>:fas-close:</md></span>
+
+**Errors:**
+If the specified event does not exist or the event does not have the specified task, an error will be thrown.
+If such a situation happens, you may double-check the task description and the event name and re-enter valid inputs.
```
Comment (Khaleelur-Rahman) [inline docs/UserGuide.md:559]: Similar consideration as previous comment.

**Review by Khaleelur-Rahman — CHANGES_REQUESTED** [review]
LGTM except for a few minor nits

File: docs/UserGuide.md [hunk 5]
```diff
@@ -480,13 +493,19 @@ Format: `delete_task td/TASK_DESCRIPTION ev/EVENT_NAME`
...
 Examples:
 * `delete_task td/Book Venue ev/NUS Career Fair 2023` deletes task `Book Venue` from the task list of the event `NUS Career Fair 2023`.
 
-### Marking a task: `mark_task`
+<box type="warning" style="background-color: #C73852; color: white">
+<span slot="icon" style="color: white;"><md>:fas-close:</md></span>
+
+**Errors:**
+If the specified event does not exist or the event does not have the specified task, an error will be thrown.
+If such a situation happens, you may double-check the task description and the event name and re-enter valid inputs.
```
[the remark this answers is not in the document]
Comment (tjch-o) [reply docs/UserGuide.md:505]: yeah in the actual website they are the same point

File: docs/UserGuide.md (hunk 1 above)
Comment (tjch-o) [reply docs/UserGuide.md:417]: nice catch! will fix

File: docs/UserGuide.md (hunk 2 above)
Comment (tjch-o) [reply docs/UserGuide.md:437]: yep will fix

**Review by Khaleelur-Rahman — APPROVED** [review]
LGTM!

**Pull request merged**